src/main/scala/net/snowflake/spark/snowflake/pushdowns/querygeneration/StringStatement.scala

```diff
@@ -11,6 +11,7 @@ import org.apache.spark.sql.catalyst.expressions.{
   Lower,
   StringLPad,
   StringRPad,
+  StringReverse,
```

Comment:
Does this mean we actually support more pushdowns in 2.3 than for 3.0?

Reply:
StringReverse is not supported from Spark 2.4, so SC needs to remove it accordingly.
So, the pushdown support is the same actually.